samples/storage_csm_powerscale_v250.yaml

```diff
@@ -280,7 +280,7 @@ spec:
       - name: dell-replication-controller-manager
         # image: Defines controller image. This shouldn't be changed
         # Allowed values: string
-        image: dellemc/dell-replication-controller:v1.3.0
+        image: dellemc/dell-replication-controller:v1.3.1
         envs:
```

Comment:
@abhi16394 , any reason behind not bumping up the config version of replication module

Reply:
@rensyct could you give more details on this for our understanding? Where is the config version?

Reply:
Line No 261 in the same file

Reply:
I believe no new config version was added, since there were nothing in there that needed change (although for consistency it would have been nice). https://github.com/dell/csm-operator/tree/main/operatorconfig/moduleconfig/replication has 1.3.0 as the latest version. I am guessing this is the reason for keeping config version as 1.3.0, but I will let @abhi16394 clarify.

Reply:
We should add new folder for replication under https://github.com/dell/csm-operator/tree/main/operatorconfig/moduleconfig/replication if config version was bumped else validation in code fails

Reply:
the replication team is not releasing the entire module
they are bumping up just the replication-controller image
Also there are no changes in replication module folder as the image is in the go file under replication.go